test/ApiTestCase.php

```diff
@@ -117,11 +139,11 @@ private function addMockMiddleware()
             new Request(
                 'GET',
                 'http://api.elifesciences.org/annotations?by='.$by.'&page='.$page.'&per-page='.$perPage.'&order='.($descendingOrder ? 'desc' : 'asc').'&use-date='.$useDate.'&access='.$access,
-                ['Accept' => new MediaType(AnnotationsClient::TYPE_ANNOTATION_LIST, 1)]
+                ['Accept' => new MediaType(AnnotationsClient::TYPE_ANNOTATION_LIST, Annotations::VERSION_ANNOTATION_LIST)]
```

Comment:
These shouldn't be used in the tests as it's no longer testing anything. For example, change `VERSION_BLOG_ARTICLE` to 1 and the tests still pass, even though it's actually not what we wanted.

Reply:
> These shouldn't be used in the tests as it's no longer testing anything. For example, change VERSION_BLOG_ARTICLE to 1 and the tests still pass, even though it's actually not what we wanted.

I'll remove it from all tests and if there are cases where it is appropriate to add it into tests we can add that later.